cypress/elements/catalog/products/products-list.js

```diff
@@ -23,13 +23,15 @@ export const PRODUCTS_LIST = {
   showFiltersButton: '[data-test-id="show-filters-button"]',
   filters: {
     filterOption: '[data-test-id="filter-option"]',
+    filterOptionField: "[class^=FilterOptionField]",
```

Comment:
If its possible avoid using classes, it would be better if you add and use data-test-id